src/app/mapping.service.ts

```diff
@@ -11,13 +11,14 @@ import { SummaryInterface } from './summary-interface';
 export class MappingService {
 
     private devServer = 'http://ves-ebi-bc.ebi.ac.uk:8081';
-    private serverUrl = 'http://localhost:8081'; // this.devServer; //
+    private SERVER_URL = 'http://localhost:8081'; // this.devServer; //
+    private BASE_URL = this.SERVER_URL + '/api/mappings';
 
-    private summaryUrl = this.serverUrl + '/api/mappings/summary';
-    private mappingsUrl = this.serverUrl + '/api/mappings';
-    private missingMappingsUrl = this.serverUrl + '/api/getmissingdiagnosismappings';
-    public exportUrl = this.serverUrl + '/api/mappings/export';
-    private uploadURL = this.serverUrl + '/api/mappings/uploads';
+    private SUMMARY_URL = this.BASE_URL + '/summary';
```

Comment:
What is the reason for the upper-casing? Just a curiosity.

Reply:
Just as a convention for constant values. https://google.github.io/styleguide/javascriptguide.xml?showone=Constants#Constants (_Use NAMES_LIKE_THIS for constant values._)

Reply:
Yes, that makes sense. Because of my nativity with Angular I do not understand how these are defined constant.

Reply:
For example, why wouldn't "devServer" be a constant?

Reply:
I see your point. That's something that can be improved. We 
could explicitly set them as readonly (I'd say the equivalent to final in Java) https://www.typescriptlang.org/docs/handbook/classes.html#readonly-modifier. I'll see if it works well and update the PR